.github/workflows/ci.yml

```diff
@@ -114,13 +114,14 @@ jobs:
       - name: Build Quarkus CLI
         run: |
           git clone https://github.com/quarkusio/quarkus.git && cd quarkus/devtools/cli && git checkout 2.7 && mvn -B -s ../../../.github/mvn-settings.xml clean install -Dquickly -Prelocations
-      - name: Install Quarkus CLI
-        run: |
-          cat <<EOF > ./quarkus-dev-cli
-          #!/bin/bash
-          java -jar $PWD/quarkus/devtools/cli/target/quarkus-cli-999-SNAPSHOT-runner.jar "\$@"
-          EOF
-          chmod +x ./quarkus-dev-cli
+# TODO https://github.com/quarkusio/registry.quarkus.io/pull/97
```

Comment:
Let's make an issue in this project out of this so we don't forget about it and we're able to plan it away, yes?

Then link it here. Other todos too.